tools/setup_perf_cluster.sh

```diff
@@ -24,8 +24,6 @@ VM_NAME=${VM_NAME:-fortio-vm}
 ISTIOCTL=${ISTIOCTL:-istioctl} # to override istioctl from outside of the path
 FORTIO_NAMESPACE=${FORTIO_NAMESPACE:-fortio} # Namespace for non istio app
 ISTIO_NAMESPACE=${ISTIO_NAMESPACE:-istio} # Namespace for istio injected app
-# Should not be set to true for perf measurement but to troubleshoot the setup
-DEBUG=false
```

Comment:
keep and use to configure the new way of configuring debug ?

Reply:
+1 for this. @zhuguihua